v2/src/plugins/markdownVariables.json

```diff
@@ -71,6 +86,28 @@
         "goEPModelsImport": "",
         "goSignIn": "SignIn",
         "pythonSignIn": "sign_in",
-        "nodeSignIn": "signIn"
+        "nodeSignIn": "signIn",
+        "nodeRecipeInitDefault": "",
+        "goRecipeInitDefault": "",
+        "pythonDefaultCallbackDefs": "",
+        "pythonRecipeInitDefault": "",
+        "reactRecipeInitDefault": ""
+    },
+    "thirdpartypasswordless": {
+        "getUserByEmailNode": "getUsersByEmail",
+        "getUserByEmailGo": "GetUsersByEmail",
+        "getUserByEmailPython": "get_users_by_email",
+        "docsLinkRecipeName": "thirdpartypasswordless",
+        "codeImportRecipeName": "thirdpartypasswordless",
+        "rid": "thirdpartypasswordless",
+        "recipeNameCapitalLetters": "ThirdPartyPasswordless",
+        "goModelName": "tpplessmodels",
+        "goGetUserById": "GetUserById",
+        "goEPModelsImport": "",
+        "reactRecipeInitDefault": "contactMethod: \"EMAIL\", // This example will work with any contactMethod\n",
+        "nodeRecipeInitDefault": "contactMethod: \"EMAIL\", // This example will work with any contactMethod\n        createAndSendCustomEmail: async (input) => console.log(input),\n        flowType: \"USER_INPUT_CODE_AND_MAGIC_LINK\", // This example will work with any flow.",
+        "goRecipeInitDefault": "",
+        "pythonDefaultCallbackDefs": "",
+        "pythonRecipeInitDefault": ""
```

Comment:
These variables are empty everywhere. Is there a need for them?

Reply:
I think they are necessary for typechecking the python/go code snippets.

Reply:
Huh.. empty link type checking? That's odd. Can you show me on a call?

Reply:
Well, they are empty because I didn't actually set their values, but I expect them to be non-empty at least for tppwless.

Reply:
cool.